src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -308,8 +309,8 @@
 
 (handlers/register-handler-fx
  :wallet.send/set-password
- (fn [{:keys [db]} [_ password]]
-   {:db (assoc-in db [:wallet :send-transaction :password] password)}))
+ (fn [{:keys [db]} [_ masked-password]]
+   {:db (assoc-in db [:wallet :send-transaction :password] (security/unmask masked-password))}))
```

Comment:
Don't we risk printing unmasked password once in app-db too? Shouldn't it be stored masked then unmasked only when directly used?

Reply:
Oh, that is a very good point! I'll address it.